<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget_12">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>    Current</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_28"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_13">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string> PACK_VOLT1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_29"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_14">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string> PACK_VOLT2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_30"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_15">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string> COL_COUNT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_31"/>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>741</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>  ADC_GAIN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_32"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_17">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string> ADC_OFFSET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_33"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="CONTROL_checkBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Controls</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>661</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_18">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> PORT :</string>
     </property>
     <property name="buddy">
      <cstring>SerialPortBox</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="SerialPortBox"/>
    <widget class="QLabel" name="label_19">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>      BaudRate</string>
     </property>
     <property name="buddy">
      <cstring>BaudRateBox</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="BaudRateBox"/>
    <widget class="QPushButton" name="start_push_button">
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="logger_checkBox">
     <property name="text">
      <string>  LOGGER</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>CELL_1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TEMP_CELL_1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>CELL_2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TEMP_CELL_2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>CELL_3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_5"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>TEMP_CELL_3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_6"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>CELL_4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_7"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>TEMP_CELL_4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_8"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>CELL_5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_9"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>TEMP_CELL_5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_10"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>CELL_6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_11"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>TEMP_CELL_6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_12"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>CELL_7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_13"/>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>TEMP_CELL_7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_14"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>CELL_8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_15"/>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>TEMP_CELL_8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_16"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>CELL_9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_17"/>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>TEMP_CELL_9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_18"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>CELL_10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_19"/>
       </item>
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>TEMP_CELL_10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_20"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>CELL_11</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_21"/>
       </item>
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>TEMP_CELL_11</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_22"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>CELL_12</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_23"/>
       </item>
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>TEMP_CELL_12</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_24"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>  Die_Temp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_25"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_18">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>      SOC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_34"/>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>20</width>
      <height>431</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>splitter</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_12</zorder>
   <zorder>layoutWidget_13</zorder>
   <zorder>layoutWidget_14</zorder>
   <zorder>layoutWidget_15</zorder>
   <zorder>line</zorder>
   <zorder>layoutWidget_16</zorder>
   <zorder>layoutWidget_17</zorder>
   <zorder>CONTROL_checkBox</zorder>
   <zorder>layoutWidget_18</zorder>
   <zorder>line_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>Configure</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_PORT"/>
    <addaction name="actionEdit_Logger"/>
    <addaction name="actionEdit_LCDs"/>
    <addaction name="actionEdit_Controls"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionCheck_Updates"/>
    <addaction name="actionDocuments"/>
   </widget>
   <addaction name="menuConfigure"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEdit_PORT">
   <property name="text">
    <string>Edit PORT ...</string>
   </property>
  </action>
  <action name="actionEdit_Logger">
   <property name="text">
    <string>Edit Logger ...</string>
   </property>
  </action>
  <action name="actionEdit_LCDs">
   <property name="text">
    <string>Edit LCDs ...</string>
   </property>
  </action>
  <action name="actionEdit_Controls">
   <property name="text">
    <string>Edit Controls ...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionCheck_Updates">
   <property name="text">
    <string>Check Updates...</string>
   </property>
  </action>
  <action name="actionDocuments">
   <property name="text">
    <string>Documents ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
